app/code/Magento/Catalog/Test/Unit/Block/Product/View/GalleryTest.php

```diff
@@ -77,6 +77,83 @@ protected function mockContext()
             ->willReturn($this->registry);
     }
 
+    public function testGetGalleryImagesJsonWithLabel(){
```

Comment:
The static tests will complain that the `{` needs to be on a new line. You should be able to run the static tests over your changes and see all the issues otherwise you can wait until Travis has finished. Though I think there will be nothing more than:

```
FILE: ...ode/Magento/Catalog/Test/Unit/Block/Product/View/GalleryTest.php
----------------------------------------------------------------------
FOUND 4 ERRORS AFFECTING 4 LINES
----------------------------------------------------------------------
  80 | ERROR | [x] Opening brace should be on a new line
  94 | ERROR | [x] Opening brace should be on a new line
 100 | ERROR | [x] Function closing brace must go on the next line
     |       |     following the body; found 1 blank lines before
     |       |     brace
 102 | ERROR | [x] Opening brace should be on a new line
----------------------------------------------------------------------
PHPCBF CAN FIX THE 4 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

Reply:
@dmanners - I have fixed the above issues, but am curious if you can point me to the correct configuration for running the static code analysis.  When I run the below command, I get more issues than what you have said. 

vendor/squizlabs/php_codesniffer/bin/phpcs app/code/Magento/Catalog/Test/Unit/Block/Product/View/GalleryTest.php

Reply:
So those results came from the build system. Not 100% sure how it was run but there are a couple of things I have done locally before.

`./vendor/bin/phpcs app/code/Magento/Catalog/Test/Unit/Block/Product/View/GalleryTest.php dev/tests/static/framework/Magento` - This will run the framework/Magento standards when running phpcs.
You could also try `bin/magento dev:tests:run static` but that runs them all.
Finally there is a way of running them only on a diff (though I have as yet got that running....) From what I understand you run `git diff origin/2.2-develop HEAD --name-only > dev/tests/static/testsuite/Magento/Test/_files/changed_files_local.txt` and this will create a list of changed files from your branch compared to another branch (in this case origin/2.2-develop). Then after that when running the suite it should only run on those files you changes. Maybe @okorshenko knows the full command for the static tests.

Reply:
There is one other option which is to setup Travis against your fork. This would then mean you could run the travis checks before providing a PR.

Reply:
@brobie ok so if you run the `git diff` as described above followed by `./vendor/bin/phpunit --config dev/tests/static/phpunit.xml.dist` that should show you the details you are looking for.

Reply:
Awesome, I'll give it a try!

Reply:
Here is the request that mimics the rules used by Magento for Code Sniffer (I think)
vendor/squizlabs/php_codesniffer/bin/phpcs --standard=dev/tests/static/framework/Magento/ruleset.xml app/code/Magento/Catalog/Test/Unit/Block/Product/View/GalleryTest.php